apps/tailwind-components/.pa11yci

```diff
@@ -23,6 +23,7 @@
         "http://localhost:3000/Icons.story",
         "http://localhost:3000/Icons.story",
         "http://localhost:3000/input/Checkbox.story",
+        "http://localhost:3000/input/File.story",
```

Comment:
I  hadn't notices this pa11yci file before, what is its purpose?

Reply:
This is the config file for pa11y. Here you can configure how to run the tests and to configure a test on a specific URL.